pkg/cri/server/sandbox_run.go

```diff
@@ -70,17 +70,6 @@ func (c *criService) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 	}
 	name := makeSandboxName(metadata)
 	log.G(ctx).WithField("podsandboxid", id).Debugf("generated id for sandbox name %q", name)
-	// Reserve the sandbox name to avoid concurrent `RunPodSandbox` request starting the
```

Comment:
We need to add TODO for sync with sbserver package.

Reply:
Is also reasonable to have a follow-up PR for sbserver?

Reply:
https://github.com/containerd/containerd/pull/7426 now tracks sbserver implementation.  @fuweid, are you alright to move forward with this one and do sbserver separately?

Reply:
Yes. Let me test it in locally and move forward~